policy-service/src/main/resources/values.yaml

```diff
@@ -33,4 +33,25 @@ resources:
 
 nodeSelector: {}
 tolerations: []
-affinity: {}
\ No newline at end of file
+affinity: {}
+
+persistence:
+  storageClass:
+  annotations: {}
+  accessMode:
+  - ReadWriteOnce
+  persistentVolumeReclaimPolicy: "Retain"
+  persistenceVolumeAccessModes:
+  - "ReadWriteOnce"
+  - "ReadOnlyMany"
+  enabled: true
+  selectors:
+    policy: "policy"
+  size: 2Gi
```

Comment:
does this need to be as high as 2Gi?
Same for the 1Gi on line 53

Reply:
This is backported from the charts values.yaml so has previously been approved as sensible

Reply:
okey doke resolved!